iterations_runners/iterations_runner.rb

```diff
@@ -31,38 +28,57 @@ def assert(cond)
     param = Integer(param)
     debug = Integer(debug) > 0
 
-    assert benchmark.end_with?(".rb")
+    if ! benchmark.end_with?(".rb")
```

Comment:
I was thinking of removing it altogether actually. None of the other runners do such a check and it will be pretty obvious what has gone wrong if you pass something with a weird filename.